litmus-portal/frontend/src/views/AnalyticsDashboard/MonitoringDashboardPage/InfoDropdown/styles.ts

```diff
@@ -0,0 +1,89 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    background: theme.palette.disabledBackground,
+  },
+  headerText: {
+    fontWeight: 'bold',
+    fontSize: '1rem',
+    fontHeight: '140%',
+    padding: theme.spacing(1.5, 0, 0, 3),
+  },
+  button: {
+    background: 'none',
+    boxShadow: 'none',
+    '&:hover': {
+      background: 'none',
```

Comment:
On hover, background and boxShadow are the same as normal. Is it still required to add here twice?